src/components/tutorial-card/index.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from 'components/tutorial-collection-cards'
 import { buildAriaLabel } from './helpers'
 import { TutorialCardProps, TutorialCardPropsWithId } from './types'
-import { ConnectedTutorialCard } from './connected-tutorial-card'
+import { TutorialCardWithBookmark } from './helpers'
```

Comment:
This can be combined with the other import from './helpers':

```tsx
import { buildAriaLabel } from './helpers'
```

GitHub won't let me do a suggestion 😭

Reply:
oh right! will update 😄